config/config_cheyenne_gnu-10.sh

```diff
@@ -30,6 +31,11 @@ module load ncarenv/1.3
 # Load these basic modules for Cheyenne
 module load cmake/3.18.2
 
+# Load Python environment
+module use /glade/p/ral/jntp/tools/miniconda3/4.8.3/modulefiles
+module load miniconda3/4.8.3
+conda activate ufs-weather-model
```

Comment:
If one does this here, one would be putting the `PATH` and `PYTHONPATH` set by these modules in the default environment of every single build script.

How does activating `ufs-weather-model` conda env translate to other/generic applications?
Is that even necessary at build time?

I would suggest doing something like this:
```
export HPC_PYTHON="miniconda3/4.8.3"

# Load Python env.
module use /glade/p/ral/jntp/tools/miniconda3/4.8.3/modulefiles
```

Add a directory with a template modulefile `modulefiles/core/hpc-miniconda3/miniconda3.lua` following the example for `python/python.lua`

Reply:
I removed the `conda activate calls`, they were not needed. Also added missing GNU-10 compatibility flags for madis. I am not going to dive into these module files, the current solution works and doesn't break anything. But I did notice some copy and paste errors in the module files when I briefly looked at them.

Reply:
Can you point out the copy/paste errors.  They should be fixed.

It works does not mean that is an acceptable solution.

Reply:
One I remember is in `modulefiles/core/python/python.lua`:
```
...
whatis("Description: GNU Compiler Family")
```
not sure if there are others.

Reply:
legit copy paste error and me being sloppy.  Thanks!
thankfully it is not a fatal error and will show false description if someone did `module show python`